src/app/components/job-card/job-card.component.scss

```diff
@@ -0,0 +1,61 @@
+@import '../../styles/colors';
+@import '../../styles/dimensions';
+@import '../../styles/typography';
+$job-card-border-radius: 8px;
+
+.home-job-item-container {
+  display: inline-block;
+  width: 180px;
+  margin: 5px;
+  background-color: $default-bg-color;
```

Comment:
Properties should be ordered display, width, margin, cursor, background-color, border-radius, box-shadow

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)